pkg/controller/realmadmin/settings_modify.go

```diff
@@ -227,6 +229,9 @@ func (c *Controller) HandleSettings() http.Handler {
 			currentRealm.AllowedTestTypes = form.AllowedTestTypes
 			currentRealm.RequireDate = form.RequireDate
 			currentRealm.AllowBulkUpload = form.AllowBulkUpload
+			if form.AllowUserReport {
+				currentRealm.EnableUserReport()
```

Comment:
Hmm - this API is weird. Is this saving the realm in the database or just setting fields on the struct?

Reply:
just setting the field on the struct - which is a bit-wise operation that seemed best encapsulated in a method